components/Form/FormField.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import AutoComplete from 'material-ui/AutoComplete';
+import ChipList from './ChipList';
+import { connect } from 'react-redux';
+import { addChip, setFieldText } from '../../actions';
+import { form } from '../../reducers';
+import { getClosestMatch } from './utility';
+import styles from './FormField.css';
+
+const onNewRequest = (value, index, formName, name, dataSourceConfig, dataSource, strict, addChip, setFieldText, limit) => {
```

Comment:
when you have this many params isn't it better to destructure and pass an object?